### ISSUE TITLE
[FEATURE] Compatibility with new gopass file format

### ISSUE THREAD

**Issue by mattwoodbury** [zeapo/Android-Password-Store#1193]
[gopass](https://github.com/gopasspw/gopass) has changed its default password file format.  See https://github.com/gopasspw/gopass/blob/master/docs/features.md#mime-secrets.  The new format is this:
```
GOPASS-SECRET-1.0
Password: foobar
User: name

more content in the body
```
`gopass` reads files in `pass` format, but new passwords created with `gopass` will now have this format. As a result, on `password-store` the password is read as "GOPASS-SECRET-1.0".  It would be great if you could have `password-store` read the yaml password when it finds "GOPASS-SECRET" in line 1.  

Thanks!

**Comment by msfjarvis** [comment]
This was requested in #1055 and implemented in #1170, please search through the issues like the template tells you to.

**Comment by mattwoodbury** [comment]
Sorry. I did try to search, but I must have done it incorrectly.  



I didn't realize that there was a new version of the app on Android that required a separate install.



-------------------------

Matthew J. Woodbury

mailto:matt@wdbry.com








---- On Fri, 06 Nov 2020 10:34:31 -0500 Harsh Shandilya <notifications@github.com> wrote ----





This was requested in https://github.com/android-password-store/Android-Password-Store/issues/1055 and implemented in https://github.com/android-password-store/Android-Password-Store/pull/1170, please search through the issues like the template tells you to.
